src/InventoryManager.java

```diff
@@ -0,0 +1,263 @@
+/**
+ * Created by robertcampos on 12/1/16.
+ */
+import java.util.*;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+public class InventoryManager {
+    // class scoped map for storing invetory items
+    private Map<String, Inventory> itemMap = new HashMap<>();
+
+    // default constructor
+    public InventoryManager() {
+
+    }
+
+    // how we run this object
+    public static void main(String[] args) {
+        InventoryManager im = new InventoryManager();
+        im.execute();
+    }
+
+    // this where all the work gets done
+    private void execute() {
+        try {
```

Comment:
Your entire execute code base is inside a try statement.  This should be avoided since you have no idea where the `Exception` will occur.  This is only slightly better than just adding `throws Exception` to the method.  You should only put the line you know may exception in the `try` block.